src/firefox/preferences.js

```diff
@@ -65,13 +69,13 @@ prefs.common = {
 };
 
 // Prefs specific to Firefox for Android.
-prefs.fennec = {
+const prefsFennec:FirefoxPreferences = {
```

Comment:
Adding the **eslint-plugin-flowtype** could definitely be a separate PR, but I'm a big fan of linting.
😻